spec/forms/candidate_interface/degree_wizard_spec.rb

```diff
@@ -954,12 +954,42 @@
       end
     end
 
-    context 'creates new degree if it does not exist' do
-      let(:degree_params) { { id: nil, award_year: '2011', application_form_id: application_form.id } }
+    context 'creates new international degree only if most fields are not blank' do
+      let(:degree_params) {
+        { id: nil, uk_or_non_uk: 'non_uk', subject: 'History', start_year: '2007', award_year: '2011', international_type: 'Bachelor of Arts',
+          university: 'University of Paris', other_grade: '94%', application_form_id: application_form.id }
+      }
+
+      it 'creates new degree entry' do
+        expect { wizard.persist! }.to change { ApplicationQualification.count }.from(1).to(2)
```

Comment:
Just curious (open question and doesn't necessarily matter) - is there a preference for this? 

I've seen two different approaches in our codebase:
`...count }.from(1).to(2)`
and
`...count }.by(1)`

Reply:
The top option seems better because it lets the dev know exactly how many records they're starting with once the let block is run line 942. In theory, they may have made a mistake and wanted to start with two records but the test would have passed. But I agree its a minimal benefit, happy to change to the other if it's preferred